slam-ws/src/nav_goal/bin/movebase_client

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+
+import nav_goal.movebase_client
+import rospy
+import sys
+
+if __name__ == '__main__':
+   if len(sys.argv) < 4:
+        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+   else:
+	try:
+		x = float(sys.argv[1])
+		y = float(sys.argv[2])
+		yaw = float(sys.argv[3])
+
+        	rospy.init_node('movebase_client_py')
+        	result = nav_goal.movebase_client.movebase_client(x, y, yaw)
+        	if result:
+            		rospy.loginfo("Nav Goal successfully sent to MoveBaseAction server!")
```

Comment:
I believe the previous comment was more accurate(goal reached) I think the code doesn't finish executing until u finish seeing the rover move and pose correctly in gazebo/viz

Reply:
Huh I'll check that, you may be right. What Erick is saying is also right tho, if we give the robot a impossible goal and it moves as close as possible and then stops, I think this code doesn't throw and error, and thus would earlier say `Goal reached!` when it hadn't actually been reached.

Reply:
Sure but that's an edge case we need to take care of ......for sure its doing more than just setting the Nav Goal its actually reaching it

Reply:
You're not wrong, I am looking into more detailed messaging if it is possible. If its not possible to tell whether it reached the target or not (at least without like unnecessary work) what message do you think is good @AbiJaya1998

Reply:
Nav goal attempt executed - Check sim to see if reached

Reply:
I think it actually may be possible to get more information out of the actionClient. Downloaded VSCode and now it's a lot easier to see what I can do lol. I'll look more into it tomorrow

Reply:
It is possible, will fix now